.github/workflows/kubeadm-e2e-crio.yaml

```diff
@@ -12,7 +12,7 @@ defaults:
 
 env:
   GOTESTSUM_VERSION: 1.7.0
-  CRIO_VERSION: 1.24.1
+  CRIO_VERSION: 1.25.1
```

Comment:
sometimes the case will fail, they fixed here: https://github.com/cri-o/cri-o/pull/6122